ui/plugins/plugins.js

```diff
@@ -16,7 +16,7 @@
 // under the License.
 (function($, cloudStack) {
   cloudStack.plugins = [
-    'quota',
-    //'testPlugin'
+    //'testPlugin',
```

Comment:
also, why not remote the testPlugin folder and the test plugin js etc. implementation?